build-farm/platform-specific-configurations/linux.sh

```diff
@@ -18,8 +18,14 @@
 SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 # shellcheck source=sbin/common/constants.sh
 source "$SCRIPT_DIR/../../sbin/common/constants.sh"
-# Bundling our own freetype can cause problems, so we skip that on linux.
-export BUILD_ARGS="${BUILD_ARGS} --skip-freetype"
+
+if [[ ("${VARIANT}" == "${BUILD_VARIANT_TEMURIN}" || "${VARIANT}" == "${BUILD_VARIANT_HOTSPOT}") && "$JAVA_FEATURE_VERSION" -ge 21 ]]; then
```

Comment:
I'm not convinced this syntax is working properly for the negative case ...
I wonder if there's really any reason to restrict this based on variant though?

Any issues with only using the `JAVA_FEATURE_VERSION` check?

(This comment applies to the same change on other platforms too of course)

Reply:
OpenJ9 already makes a conscious decision for Windows: https://github.com/adoptium/temurin-build/blob/da2408e4ea988090835f15f29cb170873cced045/build-farm/platform-specific-configurations/windows.sh#L159

Reply:
No concerns. Afaik for OpenJ9 AIX, Win, Mac already bundle freetype by default for all versions. This would start bundling on linux platforms for jdk21+.

Reply:
Thanks @pshipton @AdamBrousseau 
Since I can't reliably test building jdk21u openj9, it's probably best if that's done in a separate PR.
So I'll get this merged, and the VARIANT check can simply be updated if tests your side prove ok?

Reply:
On the other hand, if it works for Temurin it should also work for OpenJ9. I'm willing to accept the change, and if it does end up causing a problem then deal with it.

Reply:
ok sounds like a good plan, thanks @pshipton I shall update

Reply:
updated PR to remove VARIANT check

Reply:
fwiw I built most of the openj9 linux platforms with `--with-freetype=bundled` without issue. The xlinux build is still pending but shouldn't be an issue since the other platforms succeeded.

Reply:
OpenJ9 has changed the configure defaults so freetype is bundled on all platforms but linux. With this change it should be bundled on linux too for jdk21+. Once the change is accepted we can see if it's working.